### PR TITLE
New Kubernetes page in APM

### PR DESCRIPTION
**Pull request by x8a**

### CODE REVIEW

File: src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    In the **Instances** drop-down, you can select a single pod instance of your service.   
+
+    ![This is an image of the Kubernetes APM page](./images/kubernetes-header.png "Kubernetes page - Header")
+
+    </Collapser>
+    <Collapser
+    id="current-deployment"
+    title="Current deployment"
+    >
+    You can check the age of the kubernetes deployment, how many pods are available or desired, and the pods that are missing.
```
Comment (shawnj) [inline src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx:41]: Might want to add a note that the age of Deployment is when the deployment was first created.

File: src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+The Kubernetes page in APM offers the following information about your Kubernetes cluster:
+
+<CollapserGroup>
+    <Collapser
+    id="header"
+    title="Header"
+    >
+    
+    You can see your cluster name, deployment, and namespace. If you click the cluster name and the deployment, you'll get some details about each of them in a pop-up window.
```
Comment (shawnj) [inline src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx:30]: I think "slide-out window" is correct instead of "pop-up window". Maybe...

File: src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    You can check pod and their related containers details. In addition, with the **Logs** button, you can see the logs in context to that pod.
+
+    ![This is an image of the Kubernetes APM page](./images/kubernetes-pods-containers.png "Kubernetes page - Pods and Containers")
+
+    </Collapser>
+    <Collapser
+    id="activity-stream"
+    title="Activity stream"
+    >
+    You can see the stream of Kubernetes events related to that deployment and its pods.
```
Comment (shawnj) [inline src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx:68]: Maybe and additional note about clicking on the name of the Pod or Container will activate the slide out window for that Pod/Container.

File: src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    You can see the CPU and memory utilization, as well as the number of pod restarts.
+
+    ![This is an image of the Kubernetes APM page](./images/kubernetes-performance-summary.png "Kubernetes page - Performance summary")
+
+    </Collapser>
+    <Collapser
+    id="pods-containers"
+    title="Pods and Containers"
+    >
+    You can check pod and their related containers details. In addition, with the **Logs** button, you can see the logs in context to that pod.
```
Comment (shawnj) [inline src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx:59]: Maybe and additional note about clicking on the name of the Pod or Container will activate the slide out window for that Pod/Container.

File: src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+![This is an image of the Kubernetes APM page](./images/kubernetes-apm-ui.png "Kubernetes page")
+
+<figcaption>[one.newrelic.com](one.newrelic.com) > **Explorer** > Select your cluster > **Kubernetes**</figcaption>
+
+<Callout variant="tip">
+  To get a high-level overview of all your applications and services, use the [New Relic explorer](/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-one-entity-explorer).
+</Callout>
+
+## Monitor your Kubernetes cluster [#monitor-kubernetes]
```
Comment (shawnj) [inline src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx:20]: "Monitor your Deployment within your Kubernetes Cluster" ... that might be too wordy though. But the APM Page doesn't monitor the whole cluster as much as it targets a specific Deployment in the Kubernetes Cluster that is connected to your application.

Comment (x8a) [reply src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx:20]: I think it's great to be specific, so I'll change it.

File: src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+The Kubernetes page in APM offers the following information about your Kubernetes cluster:
+
+<CollapserGroup>
+    <Collapser
+    id="header"
+    title="Header"
```
[the remark this answers is not in the document]
Comment (x8a) [reply src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx:27]: I'd say we could call it Deployment summary or Deployment information, but since it doesn't have a title, I'm not sure what to name it.

File: src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx (hunk 2 above)
Comment (x8a) [reply src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx:30]: I've seen we're not very consistent through our docs about this, but I do think slide-out panel is way more descriptive.

File: src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (shawnj) [reply src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx:27]: Lets go with Deployment Summary for now.

**Comment by x8a** [comment]
This will be published on 3/1.

File: src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+metaDescription: The Kubernetes Summary page provides charts and tables that you can drill down into details about your selected app's performance.
+---
+
+The Kubernetes APM **Summary** page provides general information about the selected cluster, including the age of the current deployment, the number of pods available, CPU and memory usage, pod logs, and metric comparison.
+
+![This is an image of the Kubernetes APM page](./images/kubernetes-apm-ui.png "Kubernetes page")
+
+<figcaption>[one.newrelic.com](one.newrelic.com) > **Explorer** > Select your cluster > **Kubernetes**</figcaption>
+
+<Callout variant="tip">
```
Comment (urbiz-nr) [inline src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx:16]: Explorer, capital E

File: src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+  To get a high-level overview of all your applications and services, use the [New Relic explorer](/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-one-entity-explorer).
+</Callout>
```
Comment (urbiz-nr) [inline src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx:18]: Explorer, capital E

**Review by urbiz-nr — CHANGES_REQUESTED** [review]
Great job! Just update Explorer and you're good to go!

**Review by urbiz-nr — DISMISSED** [review]
